Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM registry.access.redhat.com/ubi8/python-38
+
+WORKDIR /archspec
+COPY . .
+
+USER root
+
+RUN dnf update -y && dnf install git python38 python3-pip -y
+RUN pip3 install poetry tox-gh-actions coverage
```

Comment:
I don't mind having these installed, but they're overkill, you don't need these to just run `archspec`